pages/content/amp-dev/documentation/guides-and-tutorials/contribute/adnetwork_integration@de.md

```diff
@@ -0,0 +1,63 @@
+## Eine `amp-ad` <a name="creating-an-amp-ad"></a> erstellen
+
+Publisher, die du als Ad Server unterstützt, betten eine von dir bereitgestellte JavaScript Bibliothek ein und platzieren auf ihren Websites diverse "Ad Snippets", die auf der JavaScript Bibliothek basieren, um Ads abzurufen und zu rendern. Da AMP den Publishern nicht erlaubt, arbiträres JavaScript auszuführen, musst du zum AMP Open Source Code beitragen und dem Tag [`amp-ad`](../../../documentation/components/reference/amp-ad.md) erlauben, Ads von deinem Ad Server anzufordern.
+
+[tip type="note"] **HINWEIS:** Mit dieser Implementierung von [`amp-ad`](../../../documentation/components/reference/amp-ad.md) kannst du herkömmliche HTML Ads **und** AMPHTML Ads anzeigen. [/tip]
+
+Der Amazon A9 Server kann beispielsweise mithilfe der folgenden Syntax aufgerufen werden:
+
+```html
+<amp-ad width="300" height="250"
+    type="a9"
+    data-aax_size="300x250"
+    data-aax_pubname="test123"
+    data-aax_src="302">
+</amp-ad>
+```
+
+Im obigen Code gibt das Attribut `type` das Werbenetzwerk an, in diesem Fall A9. Die Attribute `data-*` hängen von den Parametern ab, die der Amazon A9 Server zum Bereitstellen einer Ad vorsieht. Die Datei [`a9.js`](https://github.com/ampproject/amphtml/blob/master/ads/a9.js) zeigt dir, wie die Parameter dem JavaScript Aufruf an die URL des A9 Servers zugeordnet sind. Die entsprechenden Parameter, die vom Tag [`amp-ad`](../../../documentation/components/reference/amp-ad.md) übertragen werden, werden an die URL angehängt, um eine Ad zurückzugeben.
+
+For instructions on creating an [`amp-ad`](../../../documentation/components/reference/amp-ad.md) integration, see [Integrating ad networks into AMP](https://github.com/ampproject/amphtml/blob/master/ads/README.md).
+
+## Fast Fetch Integration erstellen <a name="creating-a-fast-fetch-integration"></a>
+
+[Fast Fetch](https://blog.amp.dev/2017/08/21/even-faster-loading-ads-in-amp/) ist ein AMP Mechanismus, der die Ad Anforderung von der Ad Antwort trennt. Dadurch können Ad Anforderungen früher im Lebenszyklus einer Seite auftreten, und Ads werden nur gerendert, wenn ihre Anzeige durch den Benutzer wahrscheinlich ist. Fast Fetch bevorzugt verifizierte AMPHTML Ads gegenüber herkömmlichen HTML Ads. Wenn in Fast Fetch eine Ad nicht validiert werden kann, wird diese Ad in ein domänenübergreifendes iframe eingeschlossen, um sie vom restlichen AMP Dokument zu isolieren. Umgekehrt wird eine validierte AMPHTML Ad direkt in die Seite geschrieben. Fast Fetch verarbeitet sowohl AMP Ads als auch nicht-AMP Ads. Für Ads, deren Validierung fehlschlägt, sind aber keine zusätzlichen Ad Anforderungen erforderlich.
+
+{{ image('/static/img/docs/ads/amphtml-ad-flow.svg', 843, 699, alt='Fast Fetch Integration flow', caption='Fast Fetch Integration flow' ) }}
+
+Um AMPHTML Ads von deinem Werbeserver aus bereitzustellen, musst du eine Fast Fetch Integration implementieren, die Folgendes umfasst:
+
+1. Supporting SSL network communication.
```

Comment:
same as above

Reply:
@sebastianbenz Thanks, translations for these are [present in GitLocalize ](https://gitlocalize.com/repo/4863/de/pages/content/amp-dev/documentation/guides-and-tutorials/contribute/adnetwork_integration.md), but my team just told me they were restarting the server when I was creating this Pull Request, which may probably be the cause the reverse parser failed. I will close this PR and recreate once again, just to be on the safe side.